---
title: 例外処理ステートメント - C# リファレンス
ms.custom: seodec18
ms.date: 07/20/2015
helpviewer_keywords:
- keywords [C#], exception handling
ms.assetid: 6d0323a3-3164-411c-9b84-a3606bd0e13c
ms.openlocfilehash: afacb21eb54c97f5108772f37f313f5ef7403142
ms.sourcegitcommit: 6b308cf6d627d78ee36dbbae8972a310ac7fd6c8
ms.translationtype: HT
ms.contentlocale: ja-JP
ms.lasthandoff: 01/23/2019
ms.locfileid: "54659593"
---
# <a name="exception-handling-statements-c-reference"></a>例外処理ステートメント (C# リファレンス)
C# では、プログラムの実行時に発生する例外と呼ばれる異常事態を処理する組み込みサポートが用意されています。 このような例外は、通常の制御フローの外部にあるコードによって処理されます。  
  
 ここでは、例外処理に関する次のトピックについて説明します。  
  
-   [throw](../../../csharp/language-reference/keywords/throw.md)  
  
-   [try-catch](../../../csharp/language-reference/keywords/try-catch.md)  
  
-   [try-finally](../../../csharp/language-reference/keywords/try-finally.md)  
  
-   [try-catch-finally](../../../csharp/language-reference/keywords/try-catch-finally.md)  
  
## <a name="see-also"></a>関連項目

- [C# リファレンス](../../../csharp/language-reference/index.md)
- [C# プログラミング ガイド](../../../csharp/programming-guide/index.md)
- [C# のキーワード](../../../csharp/language-reference/keywords/index.md)
- [ステートメントのキーワード](../../../csharp/language-reference/keywords/statement-keywords.md)
- [例外と例外処理](../../../csharp/programming-guide/exceptions/index.md)
